packages/web3torrent/.eslintrc.js

```diff
@@ -40,13 +58,14 @@ module.exports = {
     browser: true,
     es6: true
   },
-  plugins: [...baseConfig.plugins, "react-hooks", 'jest'],
+  plugins: [...baseConfig.plugins, 'jest', 'react', 'react-hooks'],
```

Comment:
Nit: I don't think you need this because of the use of `'plugin:react/recommended'` below.

```suggestion
  plugins: [...baseConfig.plugins, 'jest', 'react-hooks'],
```

Reply:
Good catch!